### DISCUSSION TITLE
目录中chapter标题前有大片空白，如何去除呢

### DISCUSSION THREAD

**Discussion by moSODOPE** [tuna/thuthesis#759] [Q&A] [answered]
当标题过长，目录中第二行标题如何去除缩进呢，如下图中红色部分
![111](https://user-images.githubusercontent.com/50304484/162354840-ba9c164a-8dca-4ae2-b0a7-45014ffe7309.jpg)
对应的.cls文件中该如何修改呢，非常感谢！
<img width="1001" alt="image" src="https://user-images.githubusercontent.com/50304484/162354970-97508298-17a1-4fde-beeb-8120c2b35380.png">

**Comment by zepinglee** [comment]
写作指南没有相关的格式要求，模板按照 LaTeX 的习惯采用了悬挂缩进的方式，也就会产生你描述的空白。
<img width="672" alt="Screen Shot 2022-04-08 at 12 33 00" src="https://user-images.githubusercontent.com/12290822/162364128-fe318c69-be86-4631-a8ae-c3edd9d68299.png">

我查了一下 Word 模板是按照左对齐的方式，但是这会导致编号不太突出。
<img width="665" alt="Screen Shot 2022-04-08 at 12 13 31" src="https://user-images.githubusercontent.com/12290822/162364076-f132d6a8-65a4-4d06-8339-548a6b0bc652.png">

我还是再询问一下研究生院的姚飞老师吧。

另外可以将你的论文目录结构发出来吗（`grep "\\\chapter\|\\\section\|\\\subsection" data/*.tex | sort`）？我想用作示例方便比较。

**Reply by moSODOPE** [reply]
主要目前我们学院要求我们目录前不能留白，如果可以请指导我如何修改.cls文件,谢谢

如果section标题不全都是很长话，那编号不突出的问题应该就不严重了吧

`\chapter{Optimal Policies in a Dual Sourcing Inventory System with Multiple Demand Classes and Supply Disruption Risks}
%======================================================================


\section{Introduction}

\section{Problem Formulation}

\section{Ordering Stage}


\section{Allocation Stage}

\section{Extensions}


\subsection{General State Transition}

\subsection{Uncertain Delivered Quantity}

\subsection{Lost Sales}

\section{Conclusion}
`

**Reply by zepinglee** [reply]
那我将续行悬挂缩进统一改成左对齐吧。

**Reply by moSODOPE** [reply]
我生成新的.cls就可以修正这个问题吗，还是要做别的什么操作

**Reply by zepinglee** [reply]
稍等，我正在处理。

**Reply by zepinglee** [reply]
@moSODOPE 你试一下 [issue#759](https://github.com/tuna/thuthesis/tree/issue%23759) 分支可以吗？需要用 xetex 编译 `.ins` 文件生成新的 `.cls`。

**Reply by moSODOPE** [reply]
非常感谢，缩进问题处理好了。
但是又出现另一问题，就是章，节标题前的空格太大了，学院老师也指出过这个问题，如下图
![741649437389_ pic_hd_副本](https://user-images.githubusercontent.com/50304484/162488966-79c07520-1782-4137-bbef-e3ea5e80b5be.jpg)
作为对比，学校word模板，章，节标题前的空间如下
<img width="567" alt="image" src="https://user-images.githubusercontent.com/50304484/162489378-3d1c2b60-19fc-44ec-963c-d2c532d6ffea.png">
我把对应地方的"\quad" 改成了"\space"，但是没见效果，还请您指导一下，或者再修改一下[issue#759](https://github.com/tuna/thuthesis/tree/issue%23759)，非常感谢

**Reply by zepinglee** [reply]
我之前向研究生院建议过标签和标题之间空 1em，只有一个空格的话区分太不明显了。研究生院原则上同意，但是实际并没写进《写作指南》，也没修改 Word 模板。既然这会给格式审查带来麻烦，我还是改成空格好了。

**Reply by zepinglee** [reply, marked as answer]
在 f333d91 修改。另外我在 64c8792 中将研究生英文版正文中的标题分隔符也改为 `\space` 了。请检查一下还有没有别的问题。
<img width="1506" alt="Screen Shot 2022-04-09 at 08 50 07" src="https://user-images.githubusercontent.com/12290822/162550035-20961b89-3324-4652-a383-5e46e0878f56.png">

**Reply by moSODOPE** [reply]
问题被完美解决了，非常感谢！

**Reply by zepinglee** [reply]
OK，那我整理一下合并进 master。

**Reply by moSODOPE** [reply]
费心了，非常感谢。

**Comment by zepinglee** [comment]
研究生院的姚飞老师发邮件说，下个版本的 Word 模板会改成悬挂缩进，跟原先的 LaTeX 模板格式一致。

如果看到更新的话记得提醒我 revert 659ad29 的修改。

**Reply by moSODOPE** [reply]
悬挂缩进啊，那看来各个学院 都要 该标准了。
好的 谢谢。

**Comment by zepinglee** [comment]
2022 年 5 月版《指南》改为悬挂缩进。

<img width="677" alt="Screen Shot 2022-05-15 at 20 40 58" src="https://user-images.githubusercontent.com/12290822/168473410-e3705e27-345b-4f6e-86f7-dae22126fc84.png">

Word 模板：
<img width="668" alt="Screen Shot 2022-05-15 at 20 41 48" src="https://user-images.githubusercontent.com/12290822/168473445-a88fab71-f622-4258-8cf1-89b0989f3f8c.png">

**Reply by moSODOPE** [reply]
好的 谢谢！
